cypress/e2e/viewOrgBounties.cy.ts

```diff
@@ -0,0 +1,52 @@
+describe('View Organization Bounties', () => {
```

Comment:
@chiragDork please rename the file and follow the file format specified on the issue, thanks